app/parse.py

```diff
@@ -15,8 +21,83 @@ class Product:
     num_of_reviews: int
 
 
-def get_all_products() -> None:
-    pass
+driver = webdriver.Chrome()
+wait = WebDriverWait(driver, 10)
+
+def accept_cookies():
+    try:
+        cookies_button = wait.until(EC.element_to_be_clickable((By.XPATH, "//button[text()='Accept']")))
+        cookies_button.click()
+        print("Accepted cookies.")
+    except:
+        print("No cookies button found.")
+
+
+def scrape_page(url, paginate=False):
+    driver.get(url)
+    accept_cookies()
+
+    products = []
+
+    if paginate:
+        while True:
+            try:
+                load_more_button = wait.until(EC.element_to_be_clickable((By.CSS_SELECTOR, ".load-more-button")))
+                ActionChains(driver).move_to_element(load_more_button).click(load_more_button).perform()
+                time.sleep(1)
+            except:
```

Comment:
Using a bare `except` is not recommended as it catches all exceptions, including system-exiting exceptions. Consider catching specific exceptions like `NoSuchElementException` for better error handling.